esrally/metrics.py

```diff
@@ -1160,11 +1160,11 @@ def format_dict(d):
     races = []
     for race in race_store(cfg).list():
         races.append([race.trial_id, time.to_iso8601(race.trial_timestamp), race.track, format_dict(race.track_params), race.challenge_name, race.car_name,
-                      format_dict(race.user_tags)])
+                      format_dict(race.user_tags), race.track_revision, race.cluster.get("team-revision")])
```

Comment:
fyi: I thought whether we might want to change the order here and group track-related properties and team-related properties but those two properties are so special that it's probably ok to leave them at the end.